File: bus-n-go-backend/src/main/kotlin/mk/ukim/finki/busngobackend/api/responses/FineResponse.kt
package mk.ukim.finki.busngobackend.api.responses

import java.sql.Timestamp

data class FineResponse(
    var id: Long,
    var iznos: Double,
    var plateno: Boolean,
    val dateCreated: Timestamp,
    var datePayed: Timestamp?,
    var dokument: String,
    val kondukter: UserResponse,
    val kontrola: ControlResponse,
    val patnik: UserResponse?,
    var telefon: String?,
    var ime: String?,
    var adresa: String?,
)

File: bus-n-go-frontend/src/app/model/responses/FineResponse.ts
import {UserResponse} from "./UserResponse";
import {ControlsResponse} from "./ControlsResponse";

export interface FineResponse {
    id: number,
    iznos: number,
    plateno: boolean,
     dateCreated: Date,
    datePayed: Date | undefined,
    dokument: string,
     kondukter: UserResponse,
     kontrola: ControlsResponse,
     patnik: UserResponse | undefined,
    telefon: string | undefined,
    ime: string | undefined,
    adresa: string | undefined,
}
